akka-docs/src/main/paradox/typed/stash.md

```diff
@@ -41,23 +41,23 @@ Java
   #stashing
 }
 
-One important thing to be aware of is that the `StashBuffer` is a buffer and stashed messages will be
+One important thing to be aware of is that the @apidoc[StashBuffer] is a buffer and stashed messages will be
 kept in memory until they are unstashed (or the actor is stopped and garbage collected). It's recommended
-to avoid stashing too many messages to avoid too much memory usage and even risking `OutOfMemoryError`
-if many actors are stashing many messages. Therefore the `StashBuffer` is bounded and the `capacity`
+to avoid stashing too many messages to avoid too much memory usage and even risking @scala[`OutOfMemoryError`]@java[@javadoc[OutOfMemoryError](java.lang.OutOfMemoryError)]
+if many actors are stashing many messages. Therefore the @apidoc[StashBuffer] is bounded and the `capacity`
 of how many messages it can hold must be specified when it's created.
 
-If you try to stash more messages than the `capacity` a `StashOverflowException` will be thrown.
-You can use `StashBuffer.isFull` before stashing a message to avoid that and take other actions, such as
+If you try to stash more messages than the `capacity` a @scala[@scaladoc[StashOverflowException](akka.actor.typed.scaladsl.StashOverflowException)]@java[@javadoc[StashOverflowException](akka.actor.typed.javadsl.StashOverflowException)] will be thrown.
+You can use @scala[@scaladoc[StashBuffer.isFull](akka.actor.typed.scaladsl.StashBuffer#isFull:Boolean)]@java[@javadoc[StashBuffer.isFull](akka.actor.typed.javadsl.StashBuffer#isFull())] before stashing a message to avoid that and take other actions, such as
```

Comment:
you can use `@apidoc` for those as well, see the docs at https://github.com/lightbend/sbt-paradox-apidoc:
```suggestion
You can use @apidoc[StashBuffer.isFull](StashBuffer) { scala="#isFull:Boolean" java="#isFull()" } before stashing a message to avoid that and take other actions, such as
```